### PR TITLE
test(suite-web): add a test for t3t1 multishare backup

### PR DESCRIPTION
**Pull request by mroz22**
so I was about to investigate #13777 which requires many manual steps and since there are no tests for t3t1 I decided that this is a great opportunity to add one. 

The problem is that I couldn't make it work. I smell some troubles with `readAndConfirmShamirMnemonicEmu`. 

I hope @grdddj will help 🙏

### CODE REVIEW

**Comment by mroz22** [comment]
quickly tested right now with (hopefully) new trezor-user-env and got this. 

image

image

**Comment by grdddj** [comment]
> quickly tested right now with (hopefully) new trezor-user-env and got this.

Right, Shamir works currently only for `T2T1`, so that is expected from https://github.com/trezor/trezor-user-env/pull/262. I will support it also for `T3T1`

**Comment by grdddj** [comment]
Shamir for `T3T1` supported in https://github.com/trezor/trezor-user-env/pull/266

**Comment by MiroslavProchazka** [comment]
Can @HajekOndrej take over this now?

**Comment by mroz22** [comment]
he has lots of stuff to do I gues, but yes, he surely can :] We maybe don't really need this test anymore (urgently), I wanted to prepare a test-driven repro for  #13777 which might have been already fixed anyway

File: packages/suite-web/e2e/tests/backup/t3t1-create-addtional-share.test.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        cy.task('swipeEmu', 'up');
+
+        // [device screen] select the number of words in your backup
+        cy.task('inputEmu', '20');
+
+        // [device screen] backup instructions
+        cy.task('swipeEmu', 'up');
+        for (const m of mnemonic.split(' ')) {
+            // [device screen] enter next word
+            cy.task('inputEmu', m);
```
Comment (tomasklim) [inline packages/suite-web/e2e/tests/backup/t3t1-create-addtional-share.test.ts]:
```suggestion
        for (const word of mnemonic.split(' ')) {
            // [device screen] enter next word
            cy.task('inputEmu', word);
```

Comment (HajekOndrej) [reply packages/suite-web/e2e/tests/backup/t3t1-create-addtional-share.test.ts]: Good idea, fixed.

**Review by tomasklim — APPROVED** [review]

**Pull request merged**